### PR TITLE
fix(tokens): process @import rule to inline font definitions in brand CSS files

### PR DESCRIPTION
**Pull request by BalbinaK**
We shouldn't use @import rule with urls due to performance issues. More info in this article: https://www.debugbear.com/blog/avoid-css-import#:~:text=Why%20does%20CSS%20%40import%20slow,can%20download%20them%20in%20parallel

Fixes [WARP-12: Use inline font-face declarations in brand CSS files](https://nmp-jira.atlassian.net/browse/WARP-12)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,9 +27,7 @@
     "./component-classes/index.d.ts"
   ],
   "dependencies": {
-    "drnm": "^0.9.0",
-    "lightningcss": "^1.19.0",
-    "@sindresorhus/slugify": "^2.2.0",
```
Comment (AnnaRybkina) [inline package.json:32]: Great catch 🙌

File: tokens/index.js [hunk 2]
```diff
@@ -4,17 +4,27 @@ import fs from 'node:fs';
...
+  const slugifiedName = slugify(tld);
+  const filename = path.join(outPath, slugifiedName) + '.css';
+
+  const fontDefinitions = `@import '@warp-ds/fonts/dist/${path.basename(filename)}';`;
   const tokens = tokenize(`./${tld}`);
-  const css = minify(tokens);
-  const filename = path.join(outPath, slugify(tld)) + '.css';
-  const fontsImport = `@import'https://assets.finn.no/pkg/@warp-ds/fonts/v1/${path.basename(filename)}';`;
-  fs.writeFileSync(filename, `${fontsImport}${css}`, 'utf-8');
+  const css = `${fontDefinitions};${tokens}`;
```
Comment (AnnaRybkina) [inline tokens/index.js:20]: While testing I couldn't see a `;` between tokens and font definitions 🤔 
But maybe it is ok and is removed by a minifier

image

Comment (BalbinaK) [reply tokens/index.js:20]: It got me at first, too! But `;` is used at the end of each line within `{}`, so this is correct. You can also see some examples on [MDN](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics) and there's no semi-colon after `{}` either 😄

**Review by AnnaRybkina — APPROVED** [review]
Looks good to me

**Pull request merged**